wordpress-mobile/AztecEditor-iOS
# Remove old code: part 2

Fixes part of #492 

**To test:**

Make sure you can build the app and the testing suite.
Run the unit tests and make sure the results match the results before the change.
